dh/csidh/fp511_amd64.s

```diff
@@ -113,33 +124,34 @@ TEXT ·mulBmiAsm(SB),NOSPLIT,$8-24
 #define MULS_MULX_512(idx, r0, r1, r2, r3, r4, r5, r6, r7, r8) \
     \ // Reduction step
```

Comment:
These are non-trivial changes and a bit hard to review without some comments as to what you did.

Reply:
pls take a look on the latest changes. I simplified the patches.